torch_geometric/nn/norm/batch_norm.py

```diff
@@ -30,18 +30,41 @@ class BatchNorm(torch.nn.Module):
             :obj:`False`, this module does not track such statistics and always
             uses batch statistics in both training and eval modes.
             (default: :obj:`True`)
+        allow_no_batch (bool, optional): If set to :obj:`True`, batches with
```

Comment:
```suggestion
        allow_single_element (bool, optional): If set to :obj:`True`, batches with
```
Can we come up with a better argument name for this, e.g., `allow_single_element`?

Reply:
Haha yes that is better, thanks